### PR TITLE
Update and rename mahdiruiz.json to _github-pages-challenge-Mrdev88.m…

### PR DESCRIPTION
**Pull request by Mrdev88**
…ahdiruiz.json

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [x] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by DEV-DIBSTER** [comment]
These files are fine (nothing is wrong with your file). However what orangc said is correct, you will need the original file.

**Comment by Mrdev88** [comment]
https://github.com/is-a-dev/register/pull/16593/commits/9090fc1a2eb73b7d8a8fec29945552bd437b00a1, is this fine

File: domains/mahdiruiz.json
[the remark this answers is not in the document]
Comment (Mrdev88) [reply domains/mahdiruiz.json:1]: I'm using mahdiruiz.is-a.dev for my github pages domain, will changing it to the other CNAME affect using this domain for all my repository's using github pages.

**Comment by Mrdev88** [comment]
https://github.com/is-a-dev/register/pull/16697

**Pull request closed without merging**